wordpress-mobile/WordPress-iOS
# Unified Prologue - Use large title on iPad

Ref #15056 

This PR applies a larger title on iPad full screen, while mantaining the original title on iPhones and iPad split screen (see below)

| | |
-- | --
<img height=500 src=https://user-images.githubusercontent.com/34376330/113437962-3dc6dc00-93ad-11eb-9d3d-f69c267da43c.png> | <img height=500 src=https://user-images.githubusercontent.com/34376330/113437992-4c14f800-93ad-11eb-838d-03384d164c98.png>



To test:

- build/run and logout if necessary
- make sure the larger title shows up on iPad full screen.
- Test on a variety of devices/conditions

## Regression Notes
1. Potential unintended areas of impact
This only changes the style of the title in the prologue. There should be no impacts on other parts of the app.

2. What I did to test those areas of impact (or what existing automated tests I relied on)


3. What automated tests I added (or what prevented me from doing so)

PR submission checklist:
